spec/abstractops.html

```diff
@@ -1446,6 +1446,7 @@ <h1>PrepareTemporalFields ( _fields_, _fieldNames_, _requiredFields_ )</h1>
           1. If _property_ is in the Property column of <emu-xref href="#table-temporal-field-requirements"></emu-xref> and there is a Conversion value in the same row, then
             1. Let _Conversion_ represent the abstract operation named by the Conversion value of the same row.
             1. Set _value_ to ? _Conversion_(_value_).
+            1. If _value_ is +∞ or -∞, throw a *RangeError* exception.
```

Comment:
would an alternative approach be to make an AO in the Conversion table that throws when it results in an infinity, and then this line wouldn't be needed?

Reply:
Good Idea! I add a new AO ToInteger and rename ToPositiveIntegerOrInfinite to ToPositiveInteger and change all the places which call them and throw on +/- Infinity. I didn't touch anywhere else which does not throw after calling them.

Reply:
PTAL